schemas/lesson.yaml

```diff
@@ -0,0 +1,29 @@
+title: Lesson
+type: object
+required:
+  - quiz
+  - video
+  - text
+properties:
+  Lessontitle:
+    description: The title of a lesson
+    type: string
+  quiz:
+    description: Test of knowledge of lesson
+    type: string
+  video:
+    description: Explanatory video of lesson
+    format: mp4
+  lessonnotes:
+    description: notes of the lesson  
+    type: string
+  videoduration:
+    description: The time in seconds/minutes/hours the video will play for.
+    type: interger
+  dateCreated: 
+    descriotion: The date on which the lesson was created.
+    type: interger
+  tutor'sname: 
```

Comment:
~tutor'sname~ `instructors`  which is an `array` of type `string` 
the values and will represent the UUID of the instructors.
remember one course can have multiple instructors